lib/Doctrine/ORM/Tools/Console/Command/ClearCache/CollectionRegionCommand.php

```diff
@@ -36,21 +37,24 @@
 class CollectionRegionCommand extends Command
 {
     /**
+     * @var SymfonyStyle
+     */
+    protected $ui;
```

Comment:
`private`

Reply:
From what I see, `ui` is not needed to be retained in internals: please convert it to a local variable wherever possible. If multiple `private` methods rely on it, please pass it around as a call parameter.

Reply:
done